bike-erp/src/components/OrderService/OrderComponent/ComponentView.tsx

```diff
@@ -1,43 +1,30 @@
-import { CardMedia, Card, CardActions, Select, FormControl, InputLabel, MenuItem } from "@material-ui/core";
 import bike_logo from "../../../assets/images/login_bike_logo.png";
-import { Box,Typography } from "@material-ui/core";
-import "./ModelView.css";
 
```

Comment:
Remove Space

Reply:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍